doc/install/plugin.rst

```diff
@@ -9,35 +9,28 @@ Linux install
 ~~~~~~~~~~~~~
 
 Obtain the Linux plugin from the link above. These instructions assume that you
-have installed Trelis in ``/opt/Trelis-16.0``. If you installed it somewhere
+have installed Cubit in ``/opt/Coreform-Cubit-2020.2``. If you installed it somewhere
 else, modify these instructions appropriately.
 
 These operations may need to be performed as the root user (sudo).
 
-1.  Unpack the tarball in ``/opt/Trelis-16.0/bin/plugins``.
-2.  Change directory to ``/opt/Trelis-16.0/bin/plugins/dagmc``.
-3.  Run the install script: ``./install.sh``.
+- Unpack the tarball in the Cubit installation folder (``/opt/Coreform-Cubit-2020.2`` by default).
```

Comment:
@bam241, do users no longer need to run an `install.sh` script to install?

Reply:
not anymore. 
the new plugin is simpler to install :)